econml/cate_interpreter/_interpreters.py

```diff
@@ -420,7 +238,24 @@ class SingleTreePolicyInterpreter(_SingleTreeInterpreter):
 
     Parameters
     ----------
-    risk_level : float or None,
+    include_uncertainty : bool, optional, default False

```

Comment:
The init arg is currently `include_model_uncertainty` even though the attribute of the class is `include_uncertainty`; if you want to keep them distinct then please correctly document the init arg's name here; otherwise pick one name and stick to it.

Reply:
done